actions/blogActions.tsx

```diff
@@ -0,0 +1,17 @@
+"use server";
+
+import { db } from "../database/database";
+import { blogs } from "../database/schemas";
+
+export const addBlog = async (title: string, content: string, slugURL: string): Promise<{ status: number; message: string }> => {
```

Comment:
We can modify this functionality with upsert logic so we can reuse this function for saving edited blogs.
https://orm.drizzle.team/learn/guides/upsert